### PR TITLE
home page for YouTube clone

### PR DESCRIPTION
**Pull request by helciodev**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+			href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+		/>
+		<link rel="stylesheet" href="style.css" />
+		<title>youtube clone</title>
+	</head>
+	<body>
+		<h1>changed</h1>
+		<nav class="navigation--menu">
+			<div class="hamburger--menu">
+				<img src="img/download.png" alt="a videoHamburger" />
```
Comment (umairarshadbutt) [inline index.html:21]: Your directory name is not matching here, `img/download.png` with `Img/download.png`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+			<div class="search--menu">
+				<form action="#">
+					<input
+						type="text"
+						placeholder="search"
+						name="search"
+						id="search"
+						class="search"
+					/>
+					<button class="search--button">
```
Comment (umairarshadbutt) [inline index.html:38]: Update the button type. `type="submit"`

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+						<i class="fa fa-search" style="color: #d3d3d3; font-size: 20px"></i>
```
Comment (umairarshadbutt) [inline index.html:39]: Remove inline CSS and move it into the external style sheet

**Review by umairarshadbutt — COMMENTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @helciodev AND @Arinpe 
Such a remarkable project you made :+1: ! 
We, however, do have some few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.


- [ ] Your directory name is not matching. Please change the directory name or update the src.
- [ ] Update the button type of search button
- [ ] Remove inline CSS and move it into the external style sheet.  ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)
- [ ] The project must be a descriptive README.md (screenshot from Pathwright 👇)
readmefile

- [ ] Pull Request has a meaningful title and a brief description (screenshot from Pathwright 👇)
pull-request


##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Comment by helciodev** [comment]
Greetings!

The required changes have being done.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Pull request merged**